Methods/Parameters/Inventory/Inventory/Program.cs

```diff
@@ -0,0 +1,314 @@
+﻿using Inventory;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Xml.Linq;
+using System.Xml.Serialization;
+
+
+
+List<Item> items = new List<Item>
+{
+new Item { Name = "Apple", Quantity = 10, Price = 10 },
+new Item { Name = "Orange", Quantity = 10, Price = 10 },
+new Item { Name = "Lemon", Quantity = 10, Price = 10 },
+new Item { Name = "Pineapple", Quantity = 10, Price = 10 },
+new Item { Name = "Carrot", Quantity = 10, Price = 10 }
+
+};
+
+
+int n = 0;
+int totalvalue;
+int choice = 0;
+bool gameon = true;
+
+
+
+//int choice = 0;
+//char choice2 = 'x';
+//int updatechoice = 0;
+//int updatedvalue = 0;
+//string item = null;
+//int Quantity = 0;
+//int Price = 0;
+
+while (gameon) {
+
+    Console.Clear();
+    MainMenu();
+    while (!int.TryParse(Console.ReadLine(), out choice))
```

Comment:
Repeated code for input validation (e.g., quantity, price) should be extracted into reusable methods.
Extract this input validation logic into a helper method (e.g., GetValidatedInt(string prompt, int minValue, int maxValue)) to avoid repeating this block across different cases.
Ex:
`int GetValidatedInput(string prompt, int maxValue)
{
    Console.WriteLine(prompt);
    int value;
    while (!int.TryParse(Console.ReadLine(), out value) || value > maxValue)
    {
        Console.WriteLine($"Invalid input. Please enter a number less than {maxValue}:");
    }
    return value;
}
`